packages/web/src/components/basic/SelectedFilters.js

```diff
@@ -82,6 +82,10 @@ class SelectedFilters extends Component {
 			if (value.location) {
 				label = `${value.location} - ${label}`;
 			}
+			// Detect if value is from a chart with chartType as custom
+			if (value && value.mainLabel && value.secondaryLabel) {
+				label = `${value.mainLabel}-${value.secondaryLabel}-${JSON.stringify(value.data)}`;
```

Comment:
@SavvyShah What if only `mainLabel` or `secondaryLabel` is present? Selected filters should handle those cases.
You should handle the `null` values as well as mentioned in Notion. Do not display `null` or `undefined` values.

Reply:
So if `mainLabel` is empty then it should be like : `${id}-${secondaryLabel}`?